src/test/java/racingcar/domain/RacingCarTest.java

```diff
@@ -0,0 +1,56 @@
+package racingcar.domain;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.NullAndEmptySource;
+
+import static org.assertj.core.api.BDDAssertions.then;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+class RacingCarTest {
+
+    private final RacingCar racingCar = new RacingCar("test");
+
+    @Test
+    void validateNameLongerThenFive() {
+        assertThrows(RuntimeException.class, () -> new RacingCar("5글자넘어요"));
+    }
+
+    @ParameterizedTest
+    @NullAndEmptySource
+    void validateNameIsEmpty(String name) {
+        assertThrows(RuntimeException.class, () -> new RacingCar(name));
+    }
+
+    @Test
+    void getPosition() {
+        then(racingCar.getPosition()).isEqualTo(0);
+    }
+
+    @Test
+    void getName() {
+        then(racingCar.getName()).isEqualTo("test");
+    }
```

Comment:
RacingCar 객체도 이름을 꺼내어 비교하지 않고 객체끼리 비교하여 검증하면 어떨까요?
```suggestion
    @Test
    void getName() {
        then(racingCar).isEqualTo(new RacingCar("test"));
    }
```